doc/nrf/ug_thingy53.rst

```diff
@@ -302,53 +279,151 @@ See `Updating Nordic Thingy:53 firmware`_ for the detailed procedures on how to
 
-* USB CDC ACM as backend for logger.
-  The logs are provided using USB CDC ACM to allow accessing them without additional hardware.
+The MCUboot bootloader is enabled by using the following Kconfigs:
 
-  Most of the Thingy:53-compatible applications and samples use only a single instance of USB CDC ACM that works as logger's backend.
-  No other USB classes are used.
-  These samples can share common USB product name, vendor ID, and product ID.
-  If a sample supports additional USB classes or more than one instance of USB CDC ACM, it must use dedicated product name, vendor ID, and product ID.
+* :kconfig:`CONFIG_BOOTLOADER_MCUBOOT`
+* :kconfig:`CONFIG_IMG_MANAGER`
+* :kconfig:`CONFIG_MCUBOOT_IMG_MANAGER`
+* :kconfig:`CONFIG_IMG_ERASE_PROGRESSIVELY`
 
-* DFU over-the-air through Simple Management Protocol over Bluetooth.
-  The application acts as GATT server and allows the connected Bluetooth Central to perform the firmware update for both application and network core.
-  The Bluetooth configuration is updated to allow quick DFU data transfer.
-  The application supports Simple Management Protocol (SMP) handlers related to:
+In addition, :kconfig:`CONFIG_UPDATEABLE_IMAGE_NUMBER` must be set to ``2`` and :kconfig:`CONFIG_MCUBOOT_IMAGE_VERSION` must be set to ``"1.0.0+0"``.
 
-  * Image management
-  * Operating System (OS) management -- used to reboot device after firmware upload is complete.
-  * Erasing settings partition -- needed to ensure that a new application would not be booted with incompatible content in the settings partition that was written by the previous application.
+See :file:`thingy53_nrf5430_cpuapp.conf` in a compatible sample for how this is done.
 
-.. tip::
-   To use your application with Thingy:53 preprogrammed bootloader, you only must set proper configuration of Partition Manager and MCUboot.
-   Other mentioned features are nice to have, but they are not mandatory.
+.. _thingy53_app_usb:
 
-   All of the Thingy:53-compatible applications in the |NCS| also support the mentioned additional features.
-   Refer to the source code of the samples and applications for examples of implementation.
+USB
+===
+
+An application compatible with the Nordic Thingy:53 is expected to enable USB CDC ACM as back-end for logging.
+The logs are provided using USB CDC ACM to allow accessing them without additional hardware.
+
+Most of the applications and samples compatible with the Nordic Thingy:53 use only a single instance of USB CDC ACM that works as the logger's back-end.
+No other USB classes are used.
+These samples can share a common USB product name, vendor ID, and product ID.
+If a sample supports additional USB classes or more than one instance of USB CDC ACM, it must use a dedicated product name, vendor ID, and product ID.
+
+The USB device stack and the CDC ACM class are enabled by using :kconfig:`CONFIG_USB_DEVICE_STACK` and :kconfig:`CONFIG_USB_CDC_ACM`.
```

Comment:
```suggestion
To enable the USB device stack and the CDC ACM class use the options :kconfig:`CONFIG_USB_DEVICE_STACK` and :kconfig:`CONFIG_USB_CDC_ACM`.
```

Reply:
Fixed.

Reply:
Combined this and the next three suggestions into a bullet point list to improve flow.